index.js

```diff
@@ -47,6 +42,77 @@ const baseRules = {
     "react/require-default-props": "error",
     "react/self-closing-comp": "error",
     "react/style-prop-object": "error",
+    "@stylistic/indent": "off", // prettier conflict
+    "@stylistic/jsx-indent-props": "off", // prettier conflict
+    "@stylistic/semi": "off", // prettier conflict
+    "@stylistic/arrow-parens": "off", // prettier conflict
+    "@stylistic/spaced-comment": "off", // @Desc conflict
+    "@stylistic/brace-style": ["error", "1tbs"],
+    "@stylistic/quote-props": ["error", "as-needed"],
+    "@stylistic/multiline-ternary": "off", // prettier conflict
+    "@stylistic/operator-linebreak": "off", // prettier conflict, unfortunatelly
+    "@stylistic/jsx-one-expression-per-line": "off",
+    "@stylistic/jsx-curly-newline": "off", // prettier conflict
+};
+
+const advancedRules = {
```

Comment:
```suggestion
const strictRules = {
```
WDYT? convention similar to **strict**-type-checked https://github.com/ibexa/eslint-config-ibexa/blob/bc172b83107bb9e5b25e11546348960f2b49a466/index.js#L154